pandas/core/strings.py

```diff
@@ -35,19 +35,32 @@
 _shared_docs = dict()
 
 
-def _get_array_list(arr, others):
+def _get_array_list(arr, others, align=True):
     from pandas.core.series import Series
```

Comment:
can you add a mini-doc string here (its an internal function so doesn't have to be full fledged, but Parameters / Returns)

Reply:
In the current version, I'm not touching anything about ``_get_array_list`` anymore. I agree that docstrings would be good, might add a draft.